app/controllers/api/functions.php

```diff
@@ -1050,7 +1050,7 @@
     ->param('entrypoint', null, new Text(1028), 'Entrypoint File.', true)
     ->param('commands', null, new Text(8192, 0), 'Build Commands.', true)
     ->param('code', [], new File(), 'Gzip file with your code package. When used with the Appwrite CLI, pass the path to your code directory, and the CLI will automatically package your code. Use a path that is within the current directory.', skipValidation: true)
-    ->param('activate', false, new Boolean(true), 'Automatically activate the deployment when it is finished building.')
+    ->param('activate', false, new Boolean(true), 'Activate the deployment when it is finished building.', skipValidation: true)
```

Comment:
Is `skipValidation` nessessary addition? Does it not work without it - what's the error?

Reply:
It doesn't actually. The issue is `('sdk.request.type', 'multipart/form-data')` actually which transforms the `false` value to an empty string (`''`) while the truthy ones are properly converted to `1`. This triggers a validation failure from the `new Boolean(true)`.